client-js/app/elements/labrad-registry.ts

```diff
@@ -614,18 +696,17 @@ export class LabradRegistry extends polymer.Base {
         this.$.pendingDialog.close();
         this.$.toastCopySuccess.show();
       }
-    }
-    else if (this.$.dragOp.innerText === 'Move') {
+    } else if (this.$.dragOp.innerText === 'Move') {
       try {
         this.$.pendingDialog.open();
-        this.$.pendingOp.innerText = "Moving...";
+        this.$.pendingOp.innerText = 'Moving...';
         switch (this.$.dragDialog.dragData['kind']) {
-          case "dir":
-            var resp = await this.socket.moveDir({path: oldPath, dir: oldName, newPath: newPath, newDir: newName});
+          case 'dir':
+            await this.socket.moveDir({path: oldPath, dir: oldName, newPath: newPath, newDir: newName});
             break;
 
-          case "key":
-            var resp = await this.socket.move({path: oldPath, key: oldName, newPath: newPath, newKey: newName});
+          case 'key':
+            await this.socket.move({path: oldPath, key: oldName, newPath: newPath, newKey: newName});
```

Comment:
It's always best to keep these kinds of formatting and other superficial changes in separate commits and/or PRs. If you have the separated out in the commit history I would suggest keeping them separate when you merge.